content/en/docs/tasks/observability/metrics/customize-metrics/index.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Customizing Istio Metrics
+description: This task shows you how to customize the Istio metrics.
+weight: 25
+keywords: [telemetry,metrics,customize]
+---
+
+This task shows you how to customize the metrics that Istio generates.
+
+Istio generates telemetry that various dashboards consume to help you visualize
+your mesh. For example, dashboards that support Istio include:
+
+* [Grafana](/docs/tasks/observability/metrics/using-istio-dashboard/)
+* [Kiali](/docs/tasks/observability/kiali/)
+* [Prometheus](/docs/tasks/observability/metrics/querying-metrics/)
+
+By default, Istio defines and generates a set of standard metrics (e.g.
+`requests_total`), but you can also customize them and create new metrics.
+
+## Custom statistics configuration
+
+Istio uses the Envoy proxy to generate metrics and provides its configuration in
+the `EnvoyFilter` at
+[`manifests/charts/istio-control/istio-discovery/templates/telemetryv2_1.6.yaml`]({{<github_blob>}}/manifests/charts/istio-control/istio-discovery/templates/telemetryv2_1.6.yaml).
+
+Configuring custom statistics involves two sections of the
+`EnvoyFilter`: `definitions` and `metrics`. The `definitions` section
```

Comment:
This is confusing because I don't see any fields named `definitions` or `metrics` (or any of the other ones mentioned in this paragraph) in the above yaml file.

Reply:
There's an Istio PR on its way, these docs are a day or two ahead of it. I added the do-not-merge while we wait for it.

Reply:
Please link this page: https://preliminary.istio.io/docs/reference/config/proxy_extensions/stats/